src/index.html

```diff
@@ -1,12 +1,46 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__container">
+    <div class="calendar">
+      <div class="calendar__container calendar__container--month-length--29">
+        <div class="calendar__day calendar__day--1 calendar__day--start-day--sun"></div>
+        <div class="calendar__day calendar__day--2"></div>
+        <div class="calendar__day calendar__day--3"></div>
+        <div class="calendar__day calendar__day--4"></div>
+        <div class="calendar__day calendar__day--5"></div>
+        <div class="calendar__day calendar__day--6"></div>
+        <div class="calendar__day calendar__day--7"></div>
+        <div class="calendar__day calendar__day--8"></div>
+        <div class="calendar__day calendar__day--9"></div>
+        <div class="calendar__day calendar__day--10"></div>
+        <div class="calendar__day calendar__day--11"></div>
+        <div class="calendar__day calendar__day--12"></div>
+        <div class="calendar__day calendar__day--13"></div>
+        <div class="calendar__day calendar__day--14"></div>
+        <div class="calendar__day calendar__day--15"></div>
+        <div class="calendar__day calendar__day--16"></div>
+        <div class="calendar__day calendar__day--17"></div>
+        <div class="calendar__day calendar__day--18"></div>
+        <div class="calendar__day calendar__day--19"></div>
+        <div class="calendar__day calendar__day--20"></div>
+        <div class="calendar__day calendar__day--21"></div>
+        <div class="calendar__day calendar__day--22"></div>
+        <div class="calendar__day calendar__day--23"></div>
+        <div class="calendar__day calendar__day--24"></div>
+        <div class="calendar__day calendar__day--25"></div>
+        <div class="calendar__day calendar__day--26"></div>
+        <div class="calendar__day calendar__day--27"></div>
+        <div class="calendar__day calendar__day--28"></div>
+        <div class="calendar__day calendar__day--29"></div>
+        <div class="calendar__day calendar__day--30"></div>
+        <div class="calendar__day calendar__day--31"></div>
```

Comment:
you should not write a modifier to each cell "calendar__day--1", you can set it in a loop for .calendar__day
@for $n from 1 through 31 {
      &:nth-child(#{$n})::before {
        content: "#{$n}";
        font-family: Arial, Helvetica, sans-serif;
        font-size: 30px;

        display: block;
        position: absolute;

        top: 50px;
        left: 50px;
        transform: translate(-50%, -50%);
      }
    }